config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/IncludeAssertionBuilder.java

```diff
@@ -35,8 +35,12 @@ public void buildAssertionElement(Element includeAssertionElement, PolicyBuilder
         }
         final String policyPath = policyGuidElement.getAttribute(POLICY_PATH);
         LOGGER.log(Level.FINE, "Looking for referenced policy include: {0}", policyPath);
-
-        final AtomicReference<Policy> includedPolicy = new AtomicReference<>(bundle.getPolicies().get(policyPath));
+        final AtomicReference<Policy> includedPolicy;
+        if (annotatedBundle != null) {
```

Comment:
Do you think that referenced policy is always found from the annotated bundle? Especially, if the referenced policy is from one of its(annotated bundle) dependencies?

Reply:
If the referenced policy is part of same module, it must be present in the AnnotatedBundle. If it is part of another module, then we check in the dependency bundle list of this bundle.